### PR TITLE
github-management: add improvements to docs

### PR DESCRIPTION
**Pull request by nikhita**
Follow up of https://github.com/kubernetes/community/pull/3294 and https://github.com/kubernetes/community/pull/3368

/cc @cblecker @spiffxp @mrbobbytables @justaugustus @rlenferink 
/assign @cblecker @spiffxp

### CODE REVIEW

File: github-management/kubernetes-repositories.md [hunk 1]
```diff
@@ -183,12 +183,13 @@ from the above procedure:
...
   * The repository **must** have an initial empty commit. The contents of the
   repo will be populated from staging by the [publishing-bot].
+  * Grant the [@kubernetes/stage-bots] team admin access to the repo.
   * Setup branch protection and enable access to the
   `stage-bots` team by adding the repo in
   [`prow/config.yaml`](https://git.k8s.io/test-infra/prow/config.yaml). See
   [kubernetes/test-infra#9292](https://github.com/kubernetes/test-infra/pull/9292)
   for an example.
-  * Once the repo has been created, ask [@nikhita] to update the
+  * Once the repo has been created, ask [@kubernetes/publishing-bot-reviewers] to update the
```
Comment (mrbobbytables) [inline github-management/kubernetes-repositories.md]: Not sure if it'd worth the effort, but would an issue template in be useful for these requests?

Comment (nikhita) [reply github-management/kubernetes-repositories.md]: At this point, it's just adding a single line. https://github.com/kubernetes/publishing-bot/pull/176

Wdyt about adding a note about adding the repo to that file? :thinking:

Comment (mrbobbytables) [reply github-management/kubernetes-repositories.md]: sgtm 👍

Comment (nikhita) [reply github-management/kubernetes-repositories.md]: Done.

**Pull request merged**